### PR TITLE
feat: results table min cell width

### PR DESCRIPTION
**Pull request by IrakliJani**
Closes: https://github.com/lightdash/lightdash/issues/3052

### Description:
- removes 100% width by default from the tables.
  - removed width from results/underlying data/sql runner tables
  - its still 100% for table viz.
- adds nowrap to table cell header labels

@PriPatel
please watch the loom and let me know what you think.

https://www.loom.com/share/11139dfd1d6945779a5b1d8979279002

🤔 🤔 🤔 🤔 🤔 

- sql runner
CleanShot 2022-09-06 at 19 01 05@2x

- underlying data
CleanShot 2022-09-06 at 18 57 59@2x

- table viz + results table
CleanShot 2022-09-06 at 18 57 54@2x

🤔 🤔 🤔 🤔 🤔

### CODE REVIEW

**Comment by TuringLovesDeathMetal** [comment]
oh god, the underlying data is _**hideous**_ 😂

**Comment by IrakliJani** [comment]
> oh god, the underlying data is _**hideous**_ 😂

yeah 😬 . should definitely be disabled for underlying data

File: packages/frontend/src/components/common/Table/Table.styles.ts [hunk 1]
```diff
@@ -152,7 +158,9 @@ export const ThActionsContainer = styled.div`
     }
 `;
 
-export const TableHeaderLabelContainer = styled.div``;
+export const TableHeaderLabelContainer = styled.div`
+    white-space: nowrap;
```
Comment (ZeRego) [inline packages/frontend/src/components/common/Table/Table.styles.ts:162]: @PriPatel fyi: You mentioned something about having the headers "shrink" if the table was close to 100% width.  With this change the table header will always be 1 line, even when using the full width table ( table viz and underlying data ).

**Comment by IrakliJani** [comment]
closing for now

**Pull request closed without merging**